projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,177 @@
+{
+  "itemGroup.slashblade": "拔刀剑",
+  "itemGroup.slashblade.desc": "抵达修罗之境",
+
+  "attribute.name.generic.reachDistance":"触及距离",
+
+  "stat.slashblade.sword_summoned": "幻影剑",
+
+  "slashblade.tooltip.killcount": "杀敌数：%1$s",
+  "slashblade.tooltip.refine": "强化等级：%1$s",
+
+  "slashblade.tooltip.material": "合成条件",
+  "slashblade.tooltip.material.requiredobjects.anvil": "合成要求场所：铁砧",
+  "slashblade.tooltip.material.killcount": "%1$s <= 杀敌数",
+  "slashblade.tooltip.material.refine": "%1$s <= 强化等级",
+  "slashblade.tooltip.material.level": "%1$s <= 等级",
+  "slashblade.tooltip.material.broken": "断刀",
+  "slashblade.tooltip.material.noscabbard": "无鞘",
+  "slashblade.tooltip.material.enchantments": "附魔",
+  
+  "item.slashblade.slashblade": "无铭「无名」",
+  "item.slashblade.slashblade.desc": "合成修理的悲剧\n纯净的容器",
+  "item.slashblade.simple.wood": "无铭刀「木偶」",
+  "item.slashblade.simple.wood.desc": "赋予刀鞘的木剑",

```

```suggestion
  "item.slashblade.simple.wood.desc": "带鞘的木刀",
```

```suggestion
  "item.slashblade.simple.wood.desc": "带刀鞘的木刀",
```
看原文是木刀而不是原版的木剑
具体请以游戏内贴图为准